src/styles/blocks/main.scss

```diff
@@ -0,0 +1,146 @@
+.main-content {
+  margin: auto;
+  min-width: 1024px;
+}
+
+.container-section {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 40px;
+}
+
+.section {
+  width: 20%;
+  min-width: 198px;
+  max-height: 406px;
+  margin: 50px 50px 0 0;
+}
+
+.section:nth-child(4n) {
+  margin-right: 0;
+}
+
+.section:nth-last-child(-n + 4) {
+  margin-top: 48px;
+}
+
+.card {
+  display: flex;
+
+  &__item {
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 200px;
+    box-sizing: border-box;
```

Comment:
It is a good tone to set box-sizing for all elements(*) and don't change it without extra need